packages/graph/hash_graph/rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "nightly-2022-09-27"
+channel = "nightly-2022-11-14"
```

Comment:
Seems that this is missing `llvm-tools-preview` https://rust-lang.github.io/rustup-components-history/aarch64-apple-darwin.html for both ARM and x86. Is that something we use?

Reply:
`llvm-tools-preview` was replaced with `llvm-tools` in a recent toolchain. I'm going to replace the makefile in `main` and use `llvm-tools` over `llvm-tools-preview`. This requires a toolchain bump for `error-stack` and `deer`